styles.css

```diff
@@ -1017,11 +1020,6 @@
     @apply text-deploy-500 decoration-deploy-500/20 hover:decoration-deploy-500;
   }
 
-  .learn-link {
```

Comment:
These are actually changes that got missed from a separate PR. We don't need the `!`s anymore now that we've got layers, and the `.learn-link` class isn't in the markup anymore (since it's only rendered in one place, so it made more sense to just move these styles there instead of having a separate `@apply` block).